administration/src/bp-modules/self-service/CardSelfServiceView.tsx

```diff
@@ -0,0 +1,141 @@
+import { Spinner } from '@blueprintjs/core'
+import React, { ReactElement, useContext, useState } from 'react'
+import styled from 'styled-components'
+
+import KoblenzLogo from '../../assets/koblenz_logo.svg'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import CardSelfServiceActivation from './CardSelfServiceActivation'
+import CardSelfServiceForm from './CardSelfServiceForm'
+import CardSelfServiceInformation from './CardSelfServiceInformation'
+import selfServiceStepInfo from './constants/selfServiceStepInfo'
+import useCardGeneratorSelfService, { CardSelfServiceStep } from './hooks/useCardGeneratorSelfService'
+
+const CenteredSpinner = styled(Spinner)`
+  position: absolute;
+  top: 45vh;
+  left: 45vw;
+`
+
+const Header = styled.div`
+  background-color: #f7f7f7;
+  display: flex;
+  justify-content: space-between;
+  justify-self: center;
+  padding: 12px;
+`
+
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+`
+
+const Container = styled.div`
+  align-self: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 500px;
+  border: 1px solid #f7f7f7;
```

Comment:
:question: We don't have a theme or similar for our colors and so on?

Reply:
no this was not implemented yet. You can create an issue for that